src/classes/storage.ts

```diff
@@ -64,6 +69,20 @@ export class Storage {
     return this.groundPlaces;
   }
 
+  /**
+   * @description Get the GroundPlaces file.
+   * @returns {GroundPlacesFile}
+   */
+  public getGroundPlacesFile(): GroundPlacesFile {
```

Comment:
i dont really understand the use case of this method and why does it need to reformat groundPlaces

Reply:
Ok alors à la base lorsque l'on passe un fichier de GroundPlaces, c'est une sorte de HashMap.
Pour faciliter le travail sur les ground places dans le SDK, je le transforme en tableau de GroundPlaces.
Tu peux voir ce code dans la méthode d'init du SDK:

````ts
public initFile(groundPlacesFile: GroundPlacesFile): void {
    const groundPlaces: GroundPlace[] = Object.entries(groundPlacesFile).map(
      ([gpuid, place]: [Gpuid, GroundPlaceFromFile]): GroundPlace => ({
        gpuid,
        ...place,
      }),
    );

    this.setGroundPlaces(groundPlaces);
 }
````
Ca serait beaucoup plus compliqué de manipuler un HashMap en interne avec toutes les méthodes de manipulation, alors j'ai fais ce choix là de plutôt partir sur un tableau.

Cette méthode fait l'effet inverse, elle re-parse les ground places dans leur format initial :) Et c'est ce fichier qui sera téléchargé à la fin par les utilisateurs finaux

Reply:
ok compris, je pensais qu'elle existait déjà.

Reply:
Il y a une autre méthode qui s'appelle `getGroundPlaces`, mais elle ne fait que récupéré les GroundPlaces au format `Array<GroundPlace>`. Elle est utilisé côté sdk/front par commodité, les tableaux sont plus simples à manipuler